components/Login/ResetPassword.js

```diff
@@ -56,12 +79,46 @@ class ResetPassword extends React.Component {
     }
 
     componentDidUpdate() {
+        const messages = defineMessages({
```

Comment:
Why do you initialize this static content on each function call? Initializing it in the constructor and storing under this is what I would suggest.